src/features/Licenses/LicensesDetailPage/index.jsx

```diff
@@ -0,0 +1,120 @@
+import React, { useState, useEffect, useRef } from 'react';
+import { Link, useParams, useHistory } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import { Button } from 'react-paragon-topaz';
+
+import { Container, Pagination } from '@edx/paragon';
+import LicenseDetailTable from 'features/Licenses/LicenseDetailTable';
+import { fetchCoursesData } from 'features/Courses/data/thunks';
+import { resetCoursesTable, updateCurrentPage } from 'features/Courses/data/slice';
+import { initialPage } from 'features/constants';
+
+import 'features/Licenses/LicensesDetailPage/index.scss';
+import { fetchLicensesData } from '../data';
+
+const LicensesDetailPage = () => {
+
+  const history = useHistory();
+  const dispatch = useDispatch();
+  const { licenseId } = useParams();
+
+  const institutionRef = useRef(undefined);
+  const [currentPage, setCurrentPage] = useState(initialPage);
+
+  const defaultLicenseInfo = {
+    licenseName: '-',
+    purchasedSeats: '-',
+    numberOfStudents: '-',
+    numberOfPendingStudents: '-',
+  };
+
+  const institution = useSelector((state) => state.main.selectedInstitution);
+  const coursesTable = useSelector((state) => state.courses.table);
+  const stateCourses = useSelector((state) => state.courses);
+  const licenseInfo = useSelector((state) => state.licenses.table.data)
+    .find((license) => license?.licenseId === parseInt(licenseId)) || defaultLicenseInfo;
+
+  const handlePagination = (targetPage) => {
+    setCurrentPage(targetPage);
+    dispatch(updateCurrentPage(targetPage));
+  };
+
+  useEffect(() => {
+    if (institution.id) {
+      dispatch(fetchCoursesData(institution.id, initialPage, stateCourses.filters, licenseId));
+      dispatch(fetchLicensesData(institution.id, initialPage, licenseId));
+    }
+
+    return () => {
+      dispatch(resetCoursesTable());
+    };
+  }, [dispatch, institution.id, licenseId]);
+
+  useEffect(() => {
+    if (institution.id !== undefined && institutionRef.current === undefined) {
+      institutionRef.current = institution.id;
+    }
+
+    if (institution.id !== institutionRef.current) {
+      history.push('/licenses');
+    }
+  }, [institution, history]);
+
+  return (
+    <Container size="xl" className="px-4">
+      <div className="d-flex justify-content-between mb-3 flex-column flex-sm-row">
+        <div className="d-flex align-items-center mb-3">
+          <Link to="/licenses" className="mr-3 link">
+            <i className="fa-solid fa-arrow-left" />
+          </Link>
+          <h3 className="h2 mb-0 course-title">License pool: {licenseInfo.licenseName}</h3>
```

Comment:
Missing text here:

"Courses for which a license can be used/assigned from this poolCourses for which a license can be used/assigned from this pool"